app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -99,7 +99,7 @@ echo "# Running registration"
 echo "#"
 
 <% if plugin_present?('katello') -%>
-if [ x$ID = xrhel ] || [ x$ID = xcentos ] || [ x$ID = xol ]; then
+if [ x$ID = xrhel ] || [ x$ID = xcentos ] || [ x$ID = xol ] || [ x$ID = xrocky ]; then
```

Comment:
I wonder if we should use `ID_LIKE` and see if `rhel` is in there.

Reply:
I thought about doing that but I felt it was safer to let the project maintainers make that decision.  The main reason I had doubts is because fedora was not already in the list and I would have expected it to be.

Reply:
@stejskalleos thoughts?

Reply:
I did some investigation (see my https://github.com/stejskalleos/os_release repo) and:
* Fedora:  doesn't have `ID_LIKE`
* CentOS `ID_LIKE="rhel fedora"`
* RHEL:  `ID_LIKE="fedora"`
* Alma Linux: `ID_LIKE="rhel centos fedora"`
* Rocky Linux: `ID_LIKE="rhel fedora"`
* Oracle Linux: `ID_LIKE="fedora"`

so we can simplify the condition to `if [ x$ID_LIKE = xfedora ]; then` and it should work

Reply:
A space like between 2 items like "rhel fedora" will break the suggested fix, I have tested  "if [ "${ID_LIKE#*fedora*}" != "$ID_LIKE" ]; then" as a working compromise and commited the change

Reply:
This compromise would also match "rhel notfedora", which may not be ideal

Reply:
Since @stejskalleos states that Fedora: doesn't have ID_LIKE, I have added  the x$ID = xfedora logic back to line 50 to avoid changing any existing functionality on Fedora hosts.